apps/python-calibration-generator/calibration_generator.py

```diff
@@ -0,0 +1,335 @@
+import argparse, numpy as np, traceback, pdb, sys, math
+from drawer import animation2D, animation3D
+from utils import RandGen, insideImage, eul2rot, saveKDRT, areAllInsideImage, insideImageMask, projectCamera, export2JSON
+from yacs.config import CfgNode as CN
```

Comment:
> yacs.config

Looks like it is a new dependency. Releases are abandoned in 2020: https://pypi.org/project/yacs/#history

Can we use `json.load()` approach from Python library?

Reply:
replaced it with yaml library, json does not work as expected.